### DISCUSSION TITLE
Default pull request title should treat dashes as dashes, if both dashes and underscores are used

### DISCUSSION THREAD

**Discussion by sj-dixon** [community/community#52735] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

My team always creates branches with names in the format of `PROJ-1234_made_some_change` where `PROJ-1234` is a ticket in JIRA.  On other projects (such as those that use conventional commits), we sometimes even do `ci/PROJ-1234/made_some_changes` or `ci/PROJ-1234_made_changes` - using multiple types of separators but always reserving dashes for jira.

So why does the default pull request title always end up as "PROJ 1234 made some changes"?

If the PR title was generated as "PROJ-1234" then I can use the github search bar to find tickets based on the JIRA number instead of always having to navigate through JIRA.

Getting people to manually change the PR title themselves is proving to be a pretty Sisyphean task.  

Even if there was a way to customize the pr title as a regex capturing group or something, that would be great.

**Comment by JoeBrunsTR** [comment]
Yes, stop making any changes to the title.  Leave dashes and underscores alone.
